src/lib/block/BuilderEdit/Calendar/feed/Module.vue

```diff
@@ -0,0 +1,168 @@
+<template>
```

Comment:
I think this should be "Record" feed, not "Module"...

Reply:
Imo it makes sense as is -- we are showing things from modules -> records, so the source is a module